core/src/test/scala/org/apache/spark/MapOutputTrackerSuite.scala

```diff
@@ -734,4 +738,112 @@ class MapOutputTrackerSuite extends SparkFunSuite with LocalSparkContext {
       tracker.stop()
     }
   }
+
+  test("SPARK-36892: Batch fetch should be enabled in some scenarios with push based shuffle") {
+    conf.set(PUSH_BASED_SHUFFLE_ENABLED, true)
+    conf.set(IS_TESTING, true)
```

Comment:
Can you rebase to latest master ? (to include Minchu's patch ?)
Specifically, we need `conf.set(SERIALIZER, "org.apache.spark.serializer.KryoSerializer")` also here to enable push based shuffle.
(Same for other tests below as well).

Reply:
Updated.